OSCompatibilityLayer.h

```diff
@@ -1,37 +1,35 @@
 #ifndef OS_COMPATABILITY_LAYER_H
 #define OS_COMPATABILITY_LAYER_H
 
-
-
+#include "Log.h"
+#include "iconvlite.h"
 #include <cstdio>
 #include <optional>
-#include <stdint.h>
 #include <set>
+#include <stdint.h>
 #include <string>
-#include "Log.h"
-#include "iconvlite.h"
 
 
 #ifndef MAX_PATH
 #define MAX_PATH 260
-#endif //MAX_PATH
+#endif // MAX_PATH
 
 
 
-//Linux specific defines
+// Linux specific defines
 #ifdef __linux
 
-//sprintf_s is not implemented on Linux. It functions similarly to the function snprintf, but the return values are different.
-//Since return values are not used, they are directly translated for now. If return values will some day be used, the compiler
-//will give an error due to sprintf_s_Linux returning void.
+// sprintf_s is not implemented on Linux. It functions similarly to the function snprintf, but the return values are
+// different. Since return values are not used, they are directly translated for now. If return values will some day be
+// used, the compiler will give an error due to sprintf_s_Linux returning void.
 #define sprintf_s sprintf_s_Linux
-void sprintf_s_Linux (char *__restrict __s, size_t __maxlen, const char *__restrict __format, ...);
+void sprintf_s_Linux(char* __restrict __s, size_t __maxlen, const char* __restrict __format, ...);
 
-//See sprintf_s_LInux
+// See sprintf_s_LInux
 #define strcpy_s strcpy_s_Linux
-void strcpy_s_Linux(char *__restrict __dest, const char *__restrict __src);
+void strcpy_s_Linux(char* __restrict __dest, const char* __restrict __src);
 
-//Very basic implementation, simply returns 0 if FILE* is not NULL
+// Very basic implementation, simply returns 0 if FILE* is not NULL
 #define fopen_s fopen_s_Linux
```

Comment:
Sadly, Vic2ToHoI4 is still using fopen_s for flags. I can look to fix that in a future cleanup.